frontend/model/notifications/periodicNotifications.js

```diff
@@ -13,16 +12,13 @@ export const PERIODIC_NOTIFICATION_TYPE = {
   MIN30: '30MIN'
 }
 
-const every1MinTimeout = { notifications: [], state: {}, delay: MINS_MILLIS }
-const every5MinTimeout = { notifications: [], state: {}, delay: 5 * MINS_MILLIS }
-const every15MinTimeout = { notifications: [], state: {}, delay: 15 * MINS_MILLIS }
-const every30MinTimeout = { notifications: [], state: {}, delay: 30 * MINS_MILLIS }
+const ephemeralNotificationState: { notifications: any[], partition: Object, clearTimeout?: Function } = { notifications: [], partition: Object.create(null) }
```

Comment:
This is such a long long, and on small screens it's kind of hard to read. Could you split this up using the brackets as guides, e.g.:

```js
const ephemeralNotificationState: {
  notifications: any[], partition: Object, clearTimeout?: Function
} = {
  notifications: [], partition: Object.create(null)
}
```